src/components/TargetResourceBrowser.js

```diff
@@ -82,13 +83,23 @@ export default function TargetResourceBrowser() {
    * then display the search input.
    **/
   const search = () => {
-    if (sourceTargetResources || sourceSearchValue) {
+    if (sourceTargetResources || sourceSearchValue || collection_error) {
+      if (collection_error) {
```

Comment:
Can you combine these 2 if statements with an && or does it break if there's no collection error?
Also again can we check against a status code instead of string?